rewards/utils/snapshot_utils.py

```diff
@@ -0,0 +1,13 @@
+from decimal import Decimal
+from config.constants.emissions import DIGG_BOOST_VAULTS
+from helpers.enums import Network
+from typing import Dict
+from rewards.snapshot.chain_snapshot import sett_snapshot
+
+
+def digg_snapshot_usd(chain: Network, block: int) -> Dict[str, Decimal]:
```

Comment:
If you return `{}` that's not going to have type of `Dict[str, Decimal]`
Think the type match here will be something like `Union[Dict[str, Decimal], Dict[None, None]`.